scripts/gcloud.sh

```diff
@@ -365,4 +359,11 @@ gcloud_reserve_static_ip_address() {
   else
     gcloud compute addresses create "${address}" --addresses="${ip}" ${gcloud_region}
   fi
+}
+
+gcloud_execute_command_instance() {
+  local name="${1}"
+  command="${2}"
```

Comment:
```suggestion
  local command="${2}"
```

Reply:
Initially I had it as local variable, but then I found some problems with serialization of quotes and by removing the local it eventually worked. Didn't look further what was the root cause.

Reply:
That should not impact how the value is handled. Weird.